torsion_target/make_torsion_target_new.py

```diff
@@ -155,7 +156,7 @@ def make_torsiondrive_target(dataset_name, torsiondrive_data, test_ff=None):
         with open('note.txt', 'w') as notefile:
             notefile.write(f'Target generated from dataset {dataset_name}, entry {entry_index}')
         # write input.mol2 file
-        qcjson_mol = qcmol.json_dict()
+        qcjson_mol = qcmol.dict(encoding='json')
```

Comment:
This is an API change in the `qcelemental` module. We will need to do this for other scripts after upgrading `qcportal` to `0.9`